### PR TITLE
Update to normalize.css v4.0.0

### PR DESCRIPTION
**Pull request by marktopper**
#19575

### CODE REVIEW

File: scss/_normalize.scss [hunk 1]
```diff
@@ -90,41 +93,49 @@ a {
...
-// Address styling not present in IE 8/9/10/11, Safari, and Chrome.
+// 1. Remove the bottom border in Firefox 39-.
+// 2. Add the correct text decoration in Chrome, Edge, IE, Opera, and Safari.
 //
 
 abbr[title] {
-  border-bottom: 1px dotted;
+  border-bottom: none; // 1
+  text-decoration: underline; // 2
+  text-decoration: underline dotted; // 2
```
Comment (marktopper) [inline scss/_normalize.scss]: This is what the original normalize looks like as well.

https://github.com/necolas/normalize.css/blob/master/normalize.css#L113

**Comment by marktopper** [comment]
What is this last violation found by @houndci-bot?

**Comment by cvrebert** [comment]
Please ignore HoundCI in this case and revert the "fixes" it wanted.
normalize.css is a special case since it's vendored.

**Comment by marktopper** [comment]
@cvrebert Reverted to be exactly as the [vendor file](https://github.com/necolas/normalize.css/blob/d311749e127ea3ea13eb900ad953270d62734f4f/normalize.css).

**Comment by cvrebert** [comment]
The `//` comment style is the one change that is desirable, because other than the initial attribution comment, [we don't want any comments in the compiled CSS](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#comments). So please use `//` comments like in the old version of the file.

**Comment by marktopper** [comment]
@cvrebert Comment style fixed for anything else than the first line.

**Comment by mdo** [comment]
Superseded by #19862 to address changes needed on our end. Thanks @marktopper for getting us started! Your commits are part of that PR <3.

**Pull request closed without merging**